.github/PULL_REQUEST_TEMPLATE.md.md

```diff
@@ -0,0 +1,41 @@
+<!--
+Please make sure you are familiar with and follow the instructions in the
+contributing guidelines (found in the CONTRIBUTING.md file).
+
+Please fill out the information below to expedite the review and (hopefully)
+merge of your pull request!
+-->
+
+## What kind of change does this PR introduce?
+
+<!-- Is it a Bug fix, feature, docs update, ... -->
```

Comment:
I'd lean away from using abbreviations, even commonly accepted ones like PR - as these forms are in English people may use translation software if they're not fluent, so making sure to just say documents or pull requests rather than docs or PR